san/graphql.py

```diff
@@ -5,45 +5,47 @@
 
 
 def execute_gql(gql_query_str):
-  headers = {}
-  if ApiConfig.api_key:
-    headers = {'authorization': "Apikey {}".format(ApiConfig.api_key)}
-
-  response = requests.post(
-    SANBASE_GQL_HOST,
-    json={
-        'query': gql_query_str},
-    headers=headers)
-
-  if response.status_code == 200:
-    return __handle_success_response__(response, gql_query_str)
-  else:
-    raise SanError(
-      "Error running query. Status code: {}.\n {}".format(
-        response.status_code,
-        gql_query_str))
+    headers = {}
+    if ApiConfig.api_key:
+        headers = {'authorization': "Apikey {}".format(ApiConfig.api_key)}
+
+    response = requests.post(
+        SANBASE_GQL_HOST,
+        json={
+            'query': gql_query_str},
```

Comment:
what formatter tool resulted in this thing?

Reply:
autopep8, guess it didn't go through this one